Sources/FoundationInternationalization/Formatting/Number/ICUNumberFormatter.swift

```diff
@@ -77,6 +78,8 @@ internal class ICUNumberFormatterBase {
                 return num == 0
             case .decimal(let num):
                 return num == 0
+            case .numericStringRepresentation(let num):
+                return Double(num)!.isZero
```

Comment:
I'd probably make it conditional instead of force unwrap since there's no guarantee that the string would always be in correct format when passed into `ICUNumberFormatterBase.Value`

Reply:
But if we do end up removing `isZero` and `doubleValue` then we won't need this at all

Reply:
I removed `isZero` and `doubleValue` in f065a2d and took these values from `FormatInput` instead.